### PR TITLE
Initial gh actions

### PR DESCRIPTION
**Pull request by jgara**
Commented out integration tests in `ci.yml` since they don't exist yet. (See pipeline doc https://github.com/eclipse-pass/main/blob/main/docs/deploy/pipeline.md)
```
#  call-tests-integration:
#    name: Run Integration Tests
#    uses: eclipse-pass/pass-dupe-checker/.github/workflows/tests-integration.yml@master
```

### CODE REVIEW

File: .github/workflows/tests-integration.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: '>=1.17.0'
+
+      - name: Run Integration Tests
+        run: go test -tags=integration ./...
```
Comment (aforward) [inline .github/workflows/tests-integration.yml:18]: Is the third dot in `...` correct?

File: .github/workflows/tests-unit.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: '>=1.17.0'
+
+      - name: Run Unit Tests
+        run: go test ./...
```
Comment (aforward) [inline .github/workflows/tests-unit.yml:18]: Same question about `./...`

**Review by aforward — CHANGES_REQUESTED** [review]
Please confirm if `./...` is correct.

File: .github/workflows/tests-integration.yml (hunk 1 above)
Comment (jgara) [reply .github/workflows/tests-integration.yml:18]: Yes. Apparently, `go test ./...` means "test all packages in a directory tree".

File: .github/workflows/tests-unit.yml (hunk 2 above)
Comment (jgara) [reply .github/workflows/tests-unit.yml:18]: Same as above. Thx

**Review by markpatton — APPROVED** [review]
Looks correct.

**Pull request merged**